Assets/Scripts/Movement.cs

```diff
@@ -7,20 +7,22 @@ public class Movement : MonoBehaviour
 {
     private CharacterController characterController;
     public Vector2 movementVector;
+
     void Start()
     {
         characterController = GetComponent<CharacterController>();
     }
+
     public void PlayerMovement(InputAction.CallbackContext context)
     {
         movementVector = context.ReadValue<Vector2>();
-        if (context.started)
-        {
-            Debug.Log("KLIK!");
-        }
-        if (context.canceled)
+    }
+
+    private void Update()
+    {
+        if (movementVector != new Vector2(0, 0)) 
         {
-            Debug.Log("canceled!");
+            characterController.Move(new Vector3(movementVector.x, 0, movementVector.y) * Time.deltaTime * 10);
         }
```

Comment:
Spróbuj zmienić grafikę octiego na zwykły quad zwrócony w stronę kamery, i dodaj efekt obrotu Octiego. Kiedy idzie w lewo, niech gracz widzi że jest on odwrócony w tamtą stronę.
Obrót wykorzystamy potem w systemie sadzenia roślinek :)

Reply:
Zmienić grafikę Octiego? Masz na myśli usunąć to "Octi Face"?